### PR TITLE
Removed AMD define in favor of ES6 import & export statements

### PR DESCRIPTION
**Pull request by mihailradkov**
This change is in preparation for replacing the static libraries with their versions from NPM

Changes in the source
- Replaced define with import statements in js/angular
- Formatted the affected code in js/angular
- Extracted angular modules in array constants for readability
- Modified js/lib/common utils to export themselves
- Added missing `var` modifiers 

Changes in the tests

- Fixed the workbench tests to work without AMD define and RequireJS
- Added Karma Webpack plugin to be able to run the tests and reused the the Webpack DEV config
- Added source map generation for easier debug in Karma
- Added `var` modifiers to some field declarations in the tests.

Jira: https://ontotext.atlassian.net/browse/GDB-3874

### CODE REVIEW

File: src/js/angular/export/controllers.js [hunk 1]
```diff
@@ -1,408 +1,407 @@
...
+                    $scope.loader = true;
+                    $http.get('repositories/' + $scope.getActiveRepository() + '/contexts').success(function (data, status, headers, config) {
+                        data.results.bindings.unshift({
+                            contextID: {
+                                type: "default",
+                                value: "The default graph"
+                            }
+                        });
+                        $scope.graphsByValue = {};
+                        for (let i in data.results.bindings) {
```
[the remark this answers is not in the document]
Comment (mihailradkov) [reply src/js/angular/export/controllers.js]: I added this `let` in August so I've totally forgotten about it while adding `var` yesterday

File: src/js/angular/graphexplore/controllers/graphs-visualizations.controller.js [hunk 2]
```diff
@@ -1,2443 +1,2444 @@
...
+        };
 
-                $scope.loading = true;
-                $http({
-                    url: 'rest/explore-graph/graph',
-                    method: 'GET',
-                    params: {
-                        query: queryString,
-                        linksLimit: $scope.saveSettings['linksLimit'],
-                        languages: !$scope.shouldShowSettings
```
[the remark this answers is not in the document]
Comment (mihailradkov) [reply src/js/angular/graphexplore/controllers/graphs-visualizations.controller.js]: I am also unsure how. Maybe some RequireJS magic?

File: src/js/angular/graphexplore/controllers/graphs-config.controller.js [hunk 3]
```diff
@@ -1,704 +1,703 @@
...
+
+    // query editor and results orientation
+    $scope.fixSizesOnHorizontalViewModeSwitch = fixSizesOnHorizontalViewModeSwitch;
+    $scope.changeViewMode = changeViewMode;
+    $scope.showHideEditor = showHideEditor;
+    $scope.focusQueryEditor = focusQueryEditor;
+    $scope.orientationViewMode = true;
+
+    // start of repository actions
+    let backendRepositoryID = $scope.getActiveRepository();
```
[the remark this answers is not in the document]
Comment (mihailradkov) [reply src/js/angular/graphexplore/controllers/graphs-config.controller.js]: Yes, after I removed requirejs I spent some time battling with errors for undefined variables so I simply started adding modifiers without taking care about code quality..

File: src/js/angular/graphexplore/controllers/graphs-visualizations.controller.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (svilenvelikov) [reply src/js/angular/graphexplore/controllers/graphs-visualizations.controller.js]: Yeah I guess that it wasn't undefined in the context of requirejs as in it's context everything was wrapped in a module function and it probably have bound the global context somehow/somewhere. Anyway. This wasn't correct IMO, so I'm glad the test caught it.

**Pull request merged**